### ISSUE TITLE
Failed to activate the Fusion package

### ISSUE THREAD

**Issue by marty80** [dudeofawesome/Fusion#1]
[Enter steps to reproduce below:]
1. pressed the "Install" button
2. the error occurs

**Atom Version**: 1.0.0
**System**: Microsoft Windows 7 Enterprise 
**Thrown From**: [Fusion](https://github.com/dudeofawesome/Fusion) package, v0.0.101
### Stack Trace

Failed to activate the Fusion package

```
At Cannot read property 'submenu' of undefined

TypeError: Cannot read property 'submenu' of undefined
  at Object.module.exports.Fusion.updateListOfBuildSystems (C:\Users\220983F\.atom\packages\Fusion\lib\fusion.coffee:194:37)
  at Object.module.exports.Fusion.activate (C:\Users\220983F\.atom\packages\Fusion\lib\fusion.coffee:45:14)
  at Package.module.exports.Package.activateNow (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:240:19)
  at C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:221:30
  at Package.module.exports.Package.measure (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:165:15)
  at Package.module.exports.Package.activate (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package.js:213:14)
  at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\package-manager.js:452:21)
  at C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\settings-view\lib\package-manager.js:362:29
  at exit (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\node_modules\settings-view\lib\package-manager.js:53:16)
  at triggerExitCallback (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\buffered-process.js:213:47)
  at C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\buffered-process.js:220:18
  at Socket.<anonymous> (C:\Users\220983F\AppData\Local\atom\app-1.0.0\resources\app.asar\src\buffered-process.js:98:18)
  at emitOne (events.js:82:20)
  at Socket.emit (events.js:166:7)
  at Pipe.close (net.js:464:12)

```
### Commands

```
     -3:29.3.0 collab:server (atom-text-editor.editor.is-focused)
     -3:24.1.0 collab:server-stop (atom-text-editor.editor.is-focused)
     -3:01.4.0 collab:server (atom-text-editor.editor.is-focused)
  7x -1:11.6.0 core:move-left (atom-text-editor.editor.is-focused)
     -1:07.8.0 core:select-right (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "seti-ui",
      "atom-monokai"
    ],
    "excludeVcsIgnoredPaths": false
  }
}
```
### Installed Packages

``` coffee
# User
atom-monokai, v0.8.0
collab, v0.1.0
Fusion, v0.0.101
git-diff-details, v0.18.0
git-go, v1.0.1
git-log, v0.4.1
git-projects, v1.14.1
git-tab-status, v1.9.2
language-cfml, v0.7.1
minimap, v4.10.0
minimap-git-diff, v4.1.3
seti-ui, v0.7.1

# Dev
No dev packages
```

**Comment by dudeofawesome** [comment]
I believe this is fixed in the newest version on APM.

Let me know if the issue continues.

**Comment by Oalecj** [comment]
Nope - still happens.

**Comment by dudeofawesome** [comment]
So, it looks like Atom's menu don't work in the exact same way on Windows as they do in Mac.  Each menu item has an "&" in it to denote the alt+character keyboard shortcut that can be used to open it.

I've fixed it in b7c11ae5388510d579b56dd76d2e80da3d151bb6
